website/public/js/controllers/filtersCtrl.js

```diff
@@ -30,6 +31,12 @@ habitrpg.controller("FiltersCtrl", ['$scope', '$rootScope', 'User', 'Shared',
       // User.save();
     };
 
+    $scope.$watch('filterQuery', function (newValue, oldValue) {
```

Comment:
Not sure. Still thinking about it. 

In general, we try to limit the number of watches going on. We already have so many :disappointed: